python/tvm/meta_schedule/tune_context.py

```diff
@@ -287,3 +287,64 @@ def notify_runner_results(
                 "Please construct TuneContext with search_strategy"
             )
         return self.search_strategy.notify_runner_results(measure_candidates, results)
+
+    def get_measure_candidate(self, mod, trace):
+        """Generate a measure candidate given an initial IR module and a trace.
+
+        Parameters
+        -----------
+        mod : IRModule
+            The initial IR module.
+        trace : Trace
+            The trace applying to the IR Module.
+
+        Returns
+        -------
+        candidate : MeasureCandidate
+            A generated candidate.
+        """
+        return _ffi_api.TuneContextGetMeasureCandidate(self, mod, trace)  # type: ignore # pylint: disable=no-member
+
+    def set_measure_candidates(self, candidates):
```

Comment:
I'm not super opinionated, but given those are private APIs in C++, shall we add an underscore prefix to them to be consistent with C++ side?

Reply:
Thanks! addressed ;)